### ISSUE TITLE
MD005 or MD007, Does not work?

### ISSUE THREAD

**Issue by ryunix** [DavidAnson/markdownlint#106]
Created the `test.md`.

```markdown
# header

- item 1
    - item 1.1
- item 2
    - item 2.1
```

```bash
$ markdownlint test.md
test.md: 4: MD007/ul-indent Unordered list indentation [Expected: 2; Actual: 4]
```

Only error on line 4 will occur.
This is same result as [mdl](https://github.com/markdownlint/markdownlint).

```bash
$ mdl test.md
test.md:4: MD007 Unordered list indentation
```

---

Fixed the `test.md`.

```markdown
# header

- item 1
  - item 1.1
- item 2
    - item 2.1
```

```bash
$ markdownlint test.md
$
```

Oops, no error. 🙀 
[mdl](https://github.com/markdownlint/markdownlint) will result in an error.

```bash
$ mdl test.md
test.md:6: MD005 Inconsistent indentation for list items at the same level
```

---

I want an MD005 error just like [mdl](https://github.com/markdownlint/markdownlint).
However, I think that MD007 error is appropriate. 🤔

**Comment by DavidAnson** [comment]
Sorry, I’ll have a look once I finish the current feature.

**Comment by ryunix** [comment]
That’s all right.  Thank you. 😸

**Comment by DavidAnson** [comment]
I have not updated the CLI to use the latest version yet, are you sure your test is using v0.8.0? (You would have needed to update the package within the global install.)

The test file above is part of every test run for v0.8.0: https://travis-ci.org/DavidAnson/markdownlint

**Comment by DavidAnson** [comment]
I think you'll find this works as you want - see below. Note that `npm` automatically nests the old version of `markdownlint` for `markdownlint-cli`, so it's necessary to remove that for the new one to be used.

```
C:\T\T>type list-indentation.md
# list-indentation

Text

- Item
    - Item {MD007}
- Item
    - Item {MD007}

Text

- Item
  - Item
- Item
    - Item {MD007}

Text

- Item
    - Item {MD007}
- Item
  - Item

C:\T\T>npm i markdownlint-cli

+ markdownlint-cli@0.7.1
added 29 packages in 2.219s

C:\T\T>node_modules\.bin\markdownlint list-indentation.md
list-indentation.md: 6: MD007/ul-indent Unordered list indentation [Expected: 2; Actual: 4]
list-indentation.md: 20: MD007/ul-indent Unordered list indentation [Expected: 2; Actual: 4]

C:\T\T>npm list markdownlint
C:\T\T
`-- markdownlint-cli@0.7.1
  `-- markdownlint@0.7.0

C:\T\T>npm i markdownlint@0.8.0

+ markdownlint@0.8.0
added 2 packages and updated 1 package in 1.282s

C:\T\T>node_modules\.bin\markdownlint list-indentation.md
list-indentation.md: 6: MD007/ul-indent Unordered list indentation [Expected: 2; Actual: 4]
list-indentation.md: 20: MD007/ul-indent Unordered list indentation [Expected: 2; Actual: 4]

C:\T\T>rd /s node_modules\markdownlint-cli\node_modules\markdownlint
node_modules\markdownlint-cli\node_modules\markdownlint, Are you sure (Y/N)? y

C:\T\T>node_modules\.bin\markdownlint list-indentation.md
list-indentation.md: 6: MD007/ul-indent Unordered list indentation [Expected: 2; Actual: 4]
list-indentation.md: 8: MD007/ul-indent Unordered list indentation [Expected: 2; Actual: 4]
list-indentation.md: 15: MD007/ul-indent Unordered list indentation [Expected: 2; Actual: 4]
list-indentation.md: 20: MD007/ul-indent Unordered list indentation [Expected: 2; Actual: 4]

C:\T\T>
```

**Comment by ryunix** [comment]
Oops. 🙀 

I'm using `markdownlint-cli`.
I thought that `markdownlint-cli` was using latest version `markdownlint`, but actually it is an old version `markdownlint`.

I confirmed that using latest version solves the problem.
Thank you very much. 😸 

Sorry to take up your time. 🙇
